oaknational/Oak-Web-Application
# [CUR-935] Add multiple curriculum partners in docx

## Description

Music year: 1988

- Added compatibility for multiple curriculum partners in the explainer section of the docx


## How to test

1. Go to https://deploy-preview-2885--oak-web-application.netlify.thenational.academy
2. Download any curriculum document with two curriculum partners
3. You should see them both in the docx
4. You should see no change in docx with a single curriculum partner 

## Screenshots

How it used to look (delete if n/a):

<img width="655" alt="Screenshot 2024-10-14 at 17 39 13" src="https://github.com/user-attachments/assets/399d720e-d198-498e-a750-28aefb30611c">

<img width="639" alt="Screenshot 2024-10-14 at 17 38 33" src="https://github.com/user-attachments/assets/cf5f73b7-6d46-40a4-9262-0d334af5ad7c">

<img width="574" alt="Screenshot 2024-10-14 at 17 39 58" src="https://github.com/user-attachments/assets/db2457e7-bcfb-4cce-b6c9-c33dba0733be">

How it should now look:

<img width="822" alt="Screenshot 2024-10-14 at 17 35 26" src="https://github.com/user-attachments/assets/94f8388a-1719-49cc-b797-0d9b205dae64">

<img width="785" alt="Screenshot 2024-10-14 at 17 36 08" src="https://github.com/user-attachments/assets/6316b81f-a92c-444e-8771-d30fd1ba9bb9">

<img width="598" alt="Screenshot 2024-10-14 at 17 41 27" src="https://github.com/user-attachments/assets/5980253a-2584-4382-aef8-4f9046605cb6">

## Checklist
